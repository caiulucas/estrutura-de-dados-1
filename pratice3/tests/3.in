9
Liam 64
Larissa 198
Rebeca 168
Louise 74
Estevao 110
Vitor 113
Alice 148
Allana 194
Emilly 97
