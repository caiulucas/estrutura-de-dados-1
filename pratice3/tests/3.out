Time 1
[L] Larissa 198
[P] Alice 148
[E] Emilly 97

Time 2
[L] Allana 194
[P] Vitor 113
[E] Louise 74

Time 3
[L] Rebeca 168
[P] Estevao 110
[E] Liam 64

